xpti/include/xpti_trace_framework.hpp

```diff
@@ -0,0 +1,311 @@
+//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//
+#pragma once
+#include <sstream>
+#include <stdint.h>
+#include <stdio.h>
```

Comment:
Why?

Reply:
Remnant from earlier debug statements, I think.. It has been taken care of.